frontend/src/pages/file-manager/components/FileListItem.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled, { css } from 'react-emotion';
+import { Checkbox } from 'carbon-components-react';
+
+const Cell = styled('td')(
+  {
+    fontSize: 20,
+    fontWeight: 400,
+    color: '#8897a2',
+  },
+  props => ({
+    padding: props.isOdd ? '26px 0 34px' : '0 0 8px',
```

Comment:
`:nth-child` would be nicer solution to styling the odd rows than passing a prop to each one: https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

Reply:
`nth-child(odd)` would work if I was styling the rows rather than the cells. However, you cannot add padding or margin to the `tr` element which is why I have done it this way to match the design. I could use it for the background colour although, if I’m doing it this way for the padding, I might as well keep it consistent and use it for the background too. (Unless you can think of a better solution of course)

Reply:
You can target the nth-child `tr` elements and then use the cascade to get to the `td` elements. I've put together an example here: https://github.com/cybsafe/phishtray/pull/34